### DISCUSSION TITLE
Manually trigger payment event.

### DISCUSSION THREAD

**Discussion by Nnamso** [community/community#19140] [Other Feature Feedback, Questions, & Ideas]
You might choose to make a manual payment for these reasons:

If payment fails, one should be able to trigger payment manually once the card details are updated.

You don't have to wait until it is triggered automatically.

A manual payment button should be added. [Pay Now]

**Comment by queenofcorgis** [comment]
Hi there! Which area of our platform is this feedback on so I can better categorize this and direct it to the proper team?

**Reply by Nnamso** [reply]
Underpayments. https://snipboard.io/pjg0y3.jpg I should be able to manually trigger payment with a button.

**Reply by queenofcorgis** [reply]
Thanks for elaborating and the screenshot @Nnamso 🎉
